examples/demo/main.go

```diff
@@ -15,10 +15,68 @@
 package main
 
 import (
-	"log"
+	"fmt"
 	"net/http"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/websocket"
+	"open-match.dev/open-match/examples/demo/bytesub"
+	"open-match.dev/open-match/internal/config"
+	"open-match.dev/open-match/internal/logging"
+)
+
+var (
+	logger = logrus.WithFields(logrus.Fields{
+		"app":       "openmatch",
+		"component": "endtoend",
```

Comment:
Other components name this more like the component path - for instance 
		"component": "app.backend.backend_service",
May be "e2e.demo"?